internal/hcs/system.go

```diff
@@ -347,11 +462,68 @@ func (computeSystem *System) PropertiesV2(ctx context.Context, types ...hcsschem
 	if propertiesJSON == "" {
 		return nil, ErrUnexpectedValue
 	}
-	properties := &hcsschema.Properties{}
-	if err := json.Unmarshal([]byte(propertiesJSON), properties); err != nil {
+	props := &hcsschema.Properties{}
+	if err := json.Unmarshal([]byte(propertiesJSON), props); err != nil {
 		return nil, makeSystemError(computeSystem, operation, err, nil)
 	}
 
+	return props, nil
+}
+
+// PropertiesV2 returns the requested container properties targeting a V2 schema container.
```

Comment:
s/container/compute system/

Reply:
huh?

Reply:
Sorry, old Unix tools syntax :)

`s/foo/bar/` generally means "replace foo with bar"

https://www.gnu.org/software/sed/manual/html_node/The-_0022s_0022-Command.html

Reply:
No I get the reference, I was confused on that this was the original comment for the function. I thought I screwed up and changed it by accident 🤣

Reply:
Changed to compute system